src/seedu/addressbook/commands/EditPhoneCommand.java

```diff
@@ -0,0 +1,46 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList.PersonNotFoundException;
+
+
+/**
+ * Edits the phone number of a person identified using it's last displayed index from the address book.
+ */
+public class EditPhoneCommand extends Command {
+
+    public static final String COMMAND_WORD = "editphone";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Edits the phone number of the person identified by the index number used in the last person listing.\n"
+            + "Parameters: INDEX NEW_PHONE_NUMBER\n"
+            + "Example: " + COMMAND_WORD + " 1 91230123";
+
+    public static final String MESSAGE_EDIT_PHONE_SUCCESS = "Edited Person: %1$s";
+
+    private String phoneNumberToChange = "";
+
+    public EditPhoneCommand(int targetVisibleIndex, String phoneNumberToChange) {
+        super(targetVisibleIndex);
+        this.phoneNumberToChange = phoneNumberToChange;
+        //System.out.println("phone number to change: " + this.phoneNumberToChange);
```

Comment:
Don't leave commented out code. Just go ahead and delete unused code. With git, you can recover any deleted code later if you really need to.